### PR TITLE
SALTO-4744: add elements with important attribute annotation to dummy

### PR DESCRIPTION
**Pull request by edenhassid**
add elements with important attribute annotation to dummy

---

_Additional context for reviewer_
added 3 more files to `records/extra`, 
final result looks like this:

Screen Shot 2023-09-07 at 14 27 31
Screen Shot 2023-09-07 at 14 27 39
Screen Shot 2023-09-07 at 14 27 47

---
_Release Notes_: 
Dummy:
* add elements with important attribute annotation to dummy

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/dummy-adapter/src/data/fixtures/fullObjWithIA.nacl.mock [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        number numAnno {
+        }
+        boolean boolAnno {
+        }
+    }
+    dummy.customFieldWithIA Active__c {
+        apiName = "Active__c"
+        externalId = false
+        label = "Active"
+      }
```
Comment (netama) [inline packages/dummy-adapter/src/data/fixtures/fullObjWithIA.nacl.mock:14]: indentation?

File: packages/dummy-adapter/src/data/fixtures/customFieldObjWithIA.nacl.mock [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    _important_values = [
+        {
+          value = "apiName"
+          indexed = false
+        },
+        {
+          value = "label"
+          indexed = true
+        },
+      ]
```
Comment (netama) [inline packages/dummy-adapter/src/data/fixtures/customFieldObjWithIA.nacl.mock:19]: looks like there's some mix of 2-space and 4-space indentation?

File: packages/dummy-adapter/src/data/fixtures/fullObjWithIA.nacl.mock [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }
+    boolean boolField {
+    }
+
+    strAnno = "ANNO"
+    numAnno = 12
+    boolAnno = true
+    _self_important_values = [
+        {
+          value = "strAnno"
```
Comment (netama) [inline packages/dummy-adapter/src/data/fixtures/fullObjWithIA.nacl.mock:28]: same re 2-vs-4 spaces

File: packages/dummy-adapter/test/generator.test.ts [hunk 4]
```diff
@@ -116,6 +126,17 @@ describe('elements generator', () => {
         expect(fragments).toHaveLength(fixture.numOfFragments)
       })
     })
+    it('should add important values annotation to relevant elements', async () => {
```
Comment (netama) [inline packages/dummy-adapter/test/generator.test.ts:129]: do we still need this test?

**Review by netama — APPROVED** [review]
LGTM with some nits

**Review by netama — APPROVED** [review]

File: packages/dummy-adapter/src/data/fixtures/fullComplexObjWithIV.nacl.mock [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    num refField {
+    }
+    "List<string>" listField {
+    }
+    dummy.FullComplexWithIV__userObj userObj {
+    }
+
+    _important_values = [
+      {
```
Comment (netama) [inline packages/dummy-adapter/src/data/fixtures/fullComplexObjWithIV.nacl.mock:19]: nit: looks like the indentation is sometimes 4 spaces and sometimes 2?

**Review by netama — APPROVED** [review]

File: packages/dummy-adapter/src/data/fixtures/fullComplexObjWithIV.nacl.mock (hunk 5 above)
Comment (edenhassid) [reply packages/dummy-adapter/src/data/fixtures/fullComplexObjWithIV.nacl.mock:19]: changing all to 2

**Pull request merged**